Example/WalletApp/PresentationLayer/Wallet/Main/MainRouter.swift

```diff
@@ -30,8 +30,8 @@ final class MainRouter {
     }
 
     func present(pushRequest: PushRequest) {
-        PushRequestModule.create(app: app, pushRequest: pushRequest)
-            .presentFullScreen(from: viewController, transparentBackground: true)
+//        PushRequestModule.create(app: app, pushRequest: pushRequest)
```

Comment:
method unused now?

Reply:
I just want to showcase w3i capabilities and limitations for dogfooding. This presents a native screen.